Add form to staff reporting procedure (#186)
TheOdinProject/top-meta: staff-guidelines.md

  - Initiatives (e.g. Gamejams, TOPathons, Fireside chats)
  - Etc.
- Communications should happen on a semi-regular cadence

> ---

Reviewer: Minor and outside of the scope of this but the headings in this doc are a little wonky, the h1s should probably be h2s

Author: Agreed! 

[8270377](https://github.com/TheOdinProject/top-meta/pull/186/commits/82703773e8d004c679e3d5dbcd725589b7d1cce1)

Author: Unfortunately the PR is going to be much harder to read now, because there was also a lot of white space at the end of a lot of lines, so I went ahead and removed them as I fixed the headings.

# Procedure for reporting a concern with an Odin Staff Member.

> Odin staff members include all members of the Core, Maintainer and Moderators Teams. Concerns can include, but should not be limited to infractions against the Social Contract, team expectations, and Discord rule infractions.

Reviewer: ```suggestion
Odin staff members include all members of the Core, Maintainer and Moderator teams. Concerns include but are not  limited to infractions against the Social Contract, team expectations, and Discord rule infractions.

```
Minor grammar nits, I think this flows a little better

Author: [8270377](https://github.com/TheOdinProject/top-meta/pull/186/commits/82703773e8d004c679e3d5dbcd725589b7d1cce1)

## How to make a report

> 1. Send a Direct Message to at least 2 non-offending Core members

Reviewer: I think these should be clarified to be either / or options and not steps

Author: [8270377](https://github.com/TheOdinProject/top-meta/pull/186/commits/82703773e8d004c679e3d5dbcd725589b7d1cce1)
